docs/presto-guide.md

```diff
@@ -1,21 +1,148 @@
-<!--
-    Licensed to the Apache Software Foundation (ASF) under one or more 
-    contributor license agreements.  See the NOTICE file distributed with
-    this work for additional information regarding copyright ownership. 
-    The ASF licenses this file to you under the Apache License, Version 2.0
-    (the "License"); you may not use this file except in compliance with 
-    the License.  You may obtain a copy of the License at
 
```

Comment:
I move the integration/presto/README.md and the /presto/Presto_Cluster_Setup_For_Carbondata.md  documents to the docs/ , merge them  and rename it to the presto-guide